tests/de/_fixtures.yaml

```diff
@@ -107,3 +107,11 @@ entities:
       out: "off"
     attributes:
       device_class: carbon_monoxide
+
+  - name: "Handy"
```

Comment:
Would call it Smartphone or Telefon

Reply:
Called it Handy since it was allready called that in another pull request and is now called that in the main branch